eng/performance/benchmark_jobs.yml

```diff
@@ -105,6 +105,15 @@ jobs:
         - ${{ if eq(parameters.osName, 'windows') }}:
           - script: (robocopy $(Build.SourcesDirectory) $(Build.SourcesDirectory)\notLocked /E /XD $(Build.SourcesDirectory)\notLocked $(Build.SourcesDirectory)\artifacts $(Build.SourcesDirectory)\.git) ^& IF %ERRORLEVEL% LEQ 1 exit 0
             displayName: Workaround to avoid locked file # https://github.com/dotnet/arcade/issues/2125
+          - script: $(CorrelationStaging)dotnet\dotnet publish -c Release -o $(CorrelationStaging)FailureReporter -f $(_Framework) -r win-${{parameters.architecture}} $(Build.SourcesDirectory)\src\tools\Reporting\FailureReporting\FailureReporting.csproj
```

Comment:
The `$(CorrelationStaging)` variable does not exist here. Instead we either use the `$(Build.SourcesDirectory)` variable directly (this is on Ubuntu) or `$(Build.SourcesDirectory)\notLocked` to work around a locked file issue in AzDO. This means that you will need to adjust where we are outputting the FailureReporter binary as well as change where we are calling dotnet from. If you look at the `install` method in `dotnet.py` you can see the algorithm we use for installing dotnet. This algorithm is used by the call to `ci_setup.py` and how we download dotnet. We do not specify a location when doing micro benchmark runs so we install in the default location, but that means we need to use that location instead of the ones we have here.